src/common/Group.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <div class="container">
+        <div class="wrapper">
+            <div class="left">
+                <img
+                    v-if="!is_team"
+                    class="contact_profile_img"
+                    src="@/assets/default-user-image.png"
+                />
+                <img
+                    v-else
+                    class="contact_profile_img"
+                    src="@/assets/group-default-image.png"
+                />
+            </div>
+            <div class="right">
+                <span class="contact_name">
+                    {{ group_name }}
+                </span>
+                <span class="recent-message">
+                    {{
+                        recent_message.length > 20
+                            ? recent_message.substring(0, 20) + '..'
+                            : recent_message
+                    }}
+                </span>
+                <span class="last-update-time">
+                    Last Update: {{ last_update }}
+                </span>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+    props: {
+        is_team: {
+            type: Boolean,
+            default: false,
+        },
+        group_name: {
+            type: String,
+            default: '',
+        },
+        contact_image: {
+            type: String,
+            default: '@/assets/default-user-image.png',
+        },
+        group_id: {
+            type: String,
+            default: null,
+        },
+        recent_message: {
+            type: String,
+            default: '',
+        },
+        last_update: {
+            type: String,
+            default: '',
+        },
+    },
+});
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Definitely look into utilising scss variable rather than custom value for common values. Nitin highly suggest this to ensure maintainability.